.github/workflows/reusable-build-test-release.yml

```diff
@@ -1716,3 +1719,16 @@ jobs:
           overwrite: true
           file_glob: true
           tag: v${{ steps.semantic.outputs.new_release_version }}
+          
+  get_workflow_conclusion:
+    if: always()
+    needs: [publish]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send logs to Skynet
+        uses: splunk/collect-ta-logs@main
+        with:
+          git_token: ${{secrets.GH_TOKEN}}
+          skynet-token: ${{ secrets.SKYNET_TOKEN }}
+          skynet-url: "https://http-inputs-services-ingest.splunkcloud.com/services/collector/event"        
```

Comment:
Can we make this as a secret as well? So we can change it easier through the https://github.com/splunk/addonfactory-sync-secrets.

Reply:
You mean move the URL to secrets?

Reply:
Yes.

Reply:
Is URL treated as secret? Are we storing other URLs to the secrets as well?

Reply:
@artemrys ?

Reply:
Sorry, I think I am fine having it here as an open string.